hopsworks-api/src/main/java/io/hops/hopsworks/api/zeppelin/rest/InterpreterDTO.java

```diff
@@ -14,6 +14,7 @@
   private String group;
   private List<LivyMsg.Session> sessions;
   private boolean notRunning;
+  private boolean defaultInterpreter;
```

Comment:
@ErmiasG wanted your opinion on how this new field is used. The default interpreter is now set in Settings.java and this field is sent to the UI and set in the noteCreateCtrl.js

Reply:
This will make the default came from two different places. I think we should stick to the zeppelin-site.xml to find what the default is.